src/uniforms/sampler.rs

```diff
@@ -69,10 +69,10 @@ pub enum MinifySamplerFilter {
     /// Takes the nearest texel from the two nearest mipmaps, and merges them.
     LinearMipmapNearest,
 
-    /// Same as `Linear` but from the nearest mipmap.
+    /// Same as `Linear`, but from the nearest mipmap.
     NearestMipmapLinear,
 
-    ///
+    /// Same as `Linear`, but from the two nearest mipmaps.
```

Comment:
This was missing, I looked it up in the GL docs, so I'm pretty sure it's correct.